package.json

```diff
@@ -4,11 +4,22 @@
     "description": "Simple HTML5 charts using the canvas element.",
     "version": "3.8.0",
     "license": "MIT",
+    "type": "module",
     "jsdelivr": "dist/chart.min.js",
     "unpkg": "dist/chart.min.js",
-    "main": "dist/chart.js",
+    "main": "dist/chart.cjs",
```

Comment:
this is not valid. If you have `"type": "module"` then `main` needs to be an ESM file. also you can remove `module` at that point as it would do nothing

Reply:
@benmccann 

New node versions with ESM support will use "exports". Old node versions without ESM support will use "main" field. So bundle in "main" field should be CommonJS.

Reply:
Fair argument. Though I'm not sure I'd want to support any version of Node that old 😆 It probably doesn't matter much one way or the other since the field basically won't be used. I do worry there's some CDN that will try to use it and we'd be better off with ESM there